queries.yaml

```diff
@@ -1592,3 +1592,11 @@ post_process:
   - fn: vectordatasource.transform.max_zoom_filter
     params:
       layers: [places]
+
+  # collision rank
+  - fn: vectordatasource.transform.add_collision_rank
```

Comment:
This will always result in collision_rank being added.

We probably only need to include collision rank when:

- Always for the POIs layer
- Only in the other layers when feature has one of `name`, `shield_text`, or `ref` (since we haven't converted some refs to shield_text)?

Perhaps this is an over optimization...

Reply:
Good point. Yes, it's probably better not to add a `collision_rank` for things which we don't think will get labelled. In 8e8a6e8 I added a filter to only add `collision_rank` if one or more of the following were true:

* It's in the `pois` layer.
* It has a name-like thing (including `name:*` language translations, `short_name`, etc...)
* It has a `ref`
* It has a `shield_text` (or `bus_shield_text`, `bicycle_shield_text`, etc...)